tests/Functional/FileFromMemoryTest.php

```diff
@@ -33,9 +33,9 @@ public function testAddFileFromMemory()
                 ];
             }, $attachments);
 
-            $this->assertContains(['name' => 'attachment[0]', 'contents' => 'File content', 'filename' => 'file1.txt'], $attachments);
-            $this->assertContains(['name' => 'attachment[1]', 'contents' => 'File content 2', 'filename' => 'file2.txt'], $attachments);
-            $this->assertContains(['name' => 'attachment[2]', 'contents' => 'Contents of a text file', 'filename' => 'text_file.txt'], $attachments);
+            $this->assertContains(['name' => 'attachment', 'contents' => 'File content', 'filename' => 'file1.txt'], $attachments);
+            $this->assertContains(['name' => 'attachment', 'contents' => 'File content 2', 'filename' => 'file2.txt'], $attachments);
+            $this->assertContains(['name' => 'attachment', 'contents' => 'Contents of a text file', 'filename' => 'text_file.txt'], $attachments);
```

Comment:
@pirogoeth Can you verify with the Mailgun API team that this is okey? I thought that we needed the array sytax. 

If this is the preferred format, please merge this PR.